src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -410,12 +412,21 @@ public void Clear()
 
             if (organelle.Definition.HasSlimeJetComponent)
             {
-                slimeJetScore += Constants.AUTO_EVO_MUCILAGE_PREDATION_SCORE;
+                // Since slime jets have only one upgrade available we don't need to check what these Upgrades have
```

Comment:
This doesn't seem futureproof so please update this code to actually check the real upgrades and confirm it is actually the mucocyst.

Reply:
Also honestly, the prey having mucocyst should subtract from the predators score, not the predator getting a bonus for having one.

Reply:
Though wouldn't that mean there's no auto-evo benefit for the species itself, meaning that it would never evolve? So there also needs to be some bonus calculation from having the mucocyst (I guess there's the "avoid predation" thing in auto-evo where this could be added).

Reply:
The avoid predation pressure works by taking 1 / the score the predator has to eat the prey, so subtracting from it actually would give a benefit.

Reply:
Talking about that score = (f(x) = 1/x for x > 0) and (f(x) = 1 for x < 0) - I don't like that function, it is not continous, get's messy for x in range [0,1] and doesn't increase the prey's score if they get better from some point so I'll change it a little bit probably for a different function
![image](https://github.com/user-attachments/assets/03aa1f56-21cb-4170-88d0-e7cc5f06f3a1)
![image](https://github.com/user-attachments/assets/b7aff160-268b-42f0-b47f-bce0dcaa4e38)